### ISSUE TITLE
[FSR] Discovery for checkbox group component not focusing on the h3

### ISSUE THREAD

**Issue by josephrlee** [department-of-veterans-affairs/va.gov-team#93136]
## Description

Enter a detailed description of the work - gherkin syntax below is acceptable.

Given: A screenreader user fills out the FSR
When: They select a screen with a checkbox group
Then: The focus should shift to the `h3` on the page like all the other pages

![image](https://github.com/user-attachments/assets/10168ef9-a99f-4f00-a5c5-a6348fec0171)

## Tasks
- [ ] Inspect why `va-checkbox-group` doesn't focus to the `h3` when a page is loaded
- [ ] Reach out to platform if needed for a fix
- [ ] If we can fix it ourselves, identify what needs to be done

## Design and Front End sync on implementation of design
- [ ] Required
- [ ] Complete
- [x] Not Required

## Acceptance criteria
- [ ] Come up with a fix for the focus issue
- [ ] Write an implementation ticket or if it's simple enough, do it in this ticket

## Testing
- [ ] N/A if non development work
- [ ] Test with a screen reader to see if focus goes to the `h3` inside the `checkbox-group-component`
- [ ] Testing passed and documented in this ticket based off the "Then" statement in the description

Testing procedure:
If additional testing steps or credentials to perform testing are needed list them here

---

## Ticket creation

<details><summary>Ticket requirements</summary>

### Required to be "Ready"
- [x] Description written
- [x] Tasks defined
- [x] Acceptance criteria written
- [ ] Testing defined
- [x] Add Labels
- [x] Attach to an Epic
- [x] Estimate Points
</details>

**Comment by OptionSelect** [comment]
Based on my testing method that I got from (this article)[https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/accessibility/focus] the initial page focus is indeed being set on the correct element, this spot is the only va-checkbox-group implementation in our app. <img width="1562" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBek1RQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--2418eba6f11bd1aacd635e81494c3af580fb539b/image.png" alt="image.png" />

FWIW as well, the screenshotted page is no longer used by our form as well so we should be good to go. Closing out and we can reopen if needed!
